### ISSUE TITLE
Start pre-review of effect size guideline

### ISSUE THREAD

**Issue by mjskay** [transparentstats/guidelines#50]
Sometime in the next two days I'm going to finish my changes to the simple effect size exemplar (i.e. addressing Chat's comments --- this is mostly already done). 

Once that is merged, I am planning to start a pull request for the effect-size branch onto master (with no intention of merging it just yet). I do not think that the effect size guideline is actually at a final state yet, but I think it is close enough to use a pull request to get us moving towards a final state and as a "pre-review": creating the pull request would allow us to comment on the effect size doc line-by-line. The nice thing about pull requests are that they aren't static, so as we address each others' comments by making changes to the effect size branch, stale comments (attached to changed content) will be resolved. In this way we can use the PR to do a "continuing review" of the doc until we think it is ready for the next step (review by two experts, etc). If that works out, perhaps using a PR for continuing review might be something we put in the review process document.

Does this sound good to others? @chatchavan @dragice @shionguha @steveharoz

**Comment by steveharoz** [comment]
I haven't done a real pass at the FAQ. I'll try to go through it tomorrow.

**Comment by chatchavan** [comment]
Sounds good to me.

**Comment by shionguha** [comment]
Sounds good.
